src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/UseLiteralsWhereAppropriate.cs

```diff
@@ -95,5 +101,12 @@ public override void Initialize(AnalysisContext context)
                 OperationKind.FieldInitializer);
             });
         }
+
+        private static bool IsConstantInterpolatedStringSupported()
+        {
+            // TODO: When constant interpolated string is supported in a stable language version (most likely C# 10), this method should be updated.
+            // The feature is currently available for preview only.
+            return false;
+        }
```

Comment:
I didn't return true for preview version based on https://github.com/dotnet/roslyn-analyzers/issues/4732#issuecomment-767891878

When this is implemented later, it will need to be abstract with language-specific implementation.

A possible another approach would be to introduce a new service that gives information about features.

I'd love to be able to do the following (or similar):

```csharp
var service = compilation.GetService<ILanguageFeatureSupportService>();
if (service.IsConstantInterpolationSupported(compilation)
{
    // do something.
}
```